### ISSUE TITLE
Old function logs build up and don't disappear

### ISSUE THREAD

**Issue by aurnik** [graphcool/console#1298]
**What is the current behavior?**
When there are a lot of logs from a function, only the oldest logs show. Also the refresh button doesn't actually do anything.

**If the current behavior is a bug, please provide the steps to reproduce and if possible screenshots.**
Create any function and have it run many times (the ones I'm looking at run ~50 times an hour).

**What is the expected behavior?**
You should be able to see the newest logs at the top, and the refresh button should refresh the logs.

**Which browser and OS are affected by this issue? Did this work in previous versions of the Console?**
Chrome 60.0.3112.113, macOS Sierra. And no I don't think this wasn't working before either.

**Comment by timsuchanek** [comment]
Thanks for reporting @aurnik ! We should definitely improve on that

**Comment by marktani** [comment]
As a heads up, the new CLI version offers a command to hook into the stream of logs for a specific function. Here's more information about the CLI beta: https://www.graph.cool/forum/t/feedback-new-cli-beta/949?u=nilan

**Comment by mbiqbal** [comment]
I have also started seeing this behaviour in web console. Earlier i had one function which showed logs till august and now yesterday another function stopped showing latest logs in console. The refresh button is not working.
@marktani It seems we have to eject the app to use the cli log streams and i don't want to do that atm. Is there any other way to see the logs?

**Comment by kbrandwijk** [comment]
I'm wondering if you run into this issue, if the new CLI logs function does display the most recent ones. @mbiqbal afaik, you can use the `-p` option of the `graphcool logs` command to connect to your project, even if it's not ejected.

**Comment by mbiqbal** [comment]
@kbrandwijk Can you guide me a bit about it? I have not used the cli before.

**Comment by mbiqbal** [comment]
@kbrandwijk From https://www.graph.cool/forum/t/feedback-new-cli-beta/949?u=nilan
Are you sure i will not lose the web console if i view the logs?
> Note: When you use the CLI to manage your project, it becomes ejected. When a project is ejected you can no longer use the Console to change types, permissions, functions or root tokens. We recommend that you try the CLI with a new project before you eject your existing projects. Read the section about Project Ejection below for more information.

**Comment by kbrandwijk** [comment]
I would ask @marktani for a confirmation, but I've tested that I can call `graphcool logs` with the new beta CLI on any project without affecting it. Don't do `graphcool pull` or anything on it though.

**Comment by mbiqbal** [comment]
@marktani Can you confirm that the project will not be ejected if i just view the logs?

**Comment by mbiqbal** [comment]
@aurnik have you got a way around it?

**Comment by aurnik** [comment]
@mbiqbal Nope, I've just been using the test run and outputting logs inside the function editor and if that's not enough I'll just use a different service for functions
